app/src/sharedTest/java/org/oppia/app/recyclerview/RecyclerViewMatcher.kt

```diff
@@ -0,0 +1,63 @@
+package org.oppia.app.recyclerview
+
+import android.content.res.Resources
+import android.view.View
+import androidx.recyclerview.widget.RecyclerView
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.TypeSafeMatcher
+
+// Reference Link: https://github.com/dannyroa/espresso-samples/blob/master/RecyclerView/app/src/androidTest/java/com/dannyroa/espresso_samples/recyclerview/RecyclerViewMatcher.java
+class RecyclerViewMatcher {
```

Comment:
Why do we need this class? Isn't it already in ``develop``? (https://github.com/oppia/oppia-android/blob/develop/app/src/sharedTest/java/org/oppia/app/recyclerview/RecyclerViewMatcher.kt).

Please remove (or merge ``develop`` into your base branch to bring this up-to-date).

Reply:
Recyclerview Matcher file is added in this PR for testcase, It is already present in develop. Once #205 is approved and update to develop this file will be removed.